manifests/j/JWLimited/Crypter/0.20.1.0/JWLimited.Crypter.installer.yaml

```
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: JWLimited.Crypter
+PackageVersion: 0.20.1.0
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: net8.0-windows10.0.22000.0\crypterv2.exe
+  PortableCommandAlias: n

```

```suggestion
  PortableCommandAlias: crypter
```
This becomes the symlink name, so please consider something closer to the application name.